hadoop-ozone/s3gateway/src/main/java/org/apache/hadoop/ozone/s3/endpoint/ObjectEndpoint.java

```diff
@@ -53,8 +53,8 @@
 import java.util.Map;
 import java.util.OptionalLong;
 
+import org.apache.commons.lang.StringUtils;
```

Comment:
```suggestion
import org.apache.commons.lang3.StringUtils;
```

`commons-lang` is not available at runtime, resulting in:

```
s3g_1       | Caused by: java.lang.NoClassDefFoundError: org/apache/commons/lang/StringUtils
s3g_1       | 	at org.apache.hadoop.ozone.s3.endpoint.ObjectEndpoint.put(ObjectEndpoint.java:180)
```